src/content/effective-dart/style.md

```diff
@@ -35,14 +35,13 @@ Identifiers come in three flavors in Dart.
 
 ### DO name types using `UpperCamelCase`
 
-{% include linter-rule-mention.md rule="camel_case_types" %}
+{% include 'linter-rule-mention.md', rule:'camel_case_types' %}
```

Comment:
The linter rule lines are styled differently (maybe you know already?):

- Before:
  > <img width="350" alt="image" src="https://github.com/dart-lang/site-www/assets/4140793/90dc4c53-da0f-47e7-be33-90bb9f8980bb">


- After:
  > <img width="350" alt="image" src="https://github.com/dart-lang/site-www/assets/4140793/00499e5e-e365-4142-9bc1-cd8f3e3e0e8a">

Reply:
Good catch, I didn't notice this one!

I went ahead and updated the include to add the styling back and accept `rules` rather than `rule`, `rule1`, and `rule2`.